avocado/utils/network/interfaces.py

```diff
@@ -259,24 +259,29 @@ def save(self, ipaddr, netmask):
         filename = "ifcfg-{}".format(self.name)
         if current_distro.name in ['rhel', 'fedora']:
             path = "/etc/sysconfig/network-scripts"
+            self._write_to_file("{}/{}".format(path, filename),
+                                {'TYPE': self.if_type,
+                                 'BOOTPROTO': 'none',
+                                 'NAME': self.name,
+                                 'DEVICE': self.name,
+                                 'ONBOOT': 'yes',
+                                 'IPADDR': ipaddr,
+                                 'NETMASK': netmask,
+                                 'IPV6INIT': 'yes',
+                                 'IPV6_AUTOCONF': 'yes',
+                                 'IPV6_DEFROUTE': 'yes'})
         elif current_distro.name == 'SuSE':
             path = "/etc/sysconfig/network"
+            self._write_to_file("{}/{}".format(path, filename),
+                                {'IPADDR': ipaddr,
+                                 'NETMASK': netmask,
+                                 'IPV6INIT': 'yes',
+                                 'IPV6_AUTOCONF': 'yes',
+                                 'IPV6_DEFROUTE': 'yes'})
```

Comment:
Hi @vaishnavibhat , thanks for that. I just have one small suggestion:

Since this has a potencial grow as we add new distros and there are a set of common parameters I would suggest something like that:

```python
config_dict = {a common dict here}
if current_distro.name in ['rhel', 'fedora']:
    config_dict.update({only specific for rhel, fedora})
elif current_distro.name == 'SuSe':
    config_dict.update({only specific for suse})
self._write_to_file(filename, config_dict)
```
I think that we could save a few lines here. What do you think ?